org.trelby.Trelby.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: org.trelby.Trelby
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: trelby
+finish-args:
+  - --socket=cups
+  - --share=ipc
+  - --socket=fallback-x11 #Known bug in wxWidgets, it is hardcoded to use X11 no matter what, for now it will only support X or XWayland. https://github.com/wxWidgets/Phoenix/issues/2557
```

Comment:
this should be `x11`

Reply:
Sorry about that, I assumed it didn't matter because it worked with either setting be it x11 or x11-fallback